### PR TITLE
[LC-799] remove message_code.Request.status and its handler.

### PR DESCRIPTION
**Pull request by windies21**
* remove message_code.Request.status and its handler. 
* remove GetScoreStatus in gRPC interface.

### CODE REVIEW

File: loopchain/channel/channel_inner_service.py [hunk 1]
```diff
@@ -25,7 +13,8 @@
 
 from loopchain import configure as conf
 from loopchain import utils as util
-from loopchain.baseservice import BroadcastCommand, BroadcastScheduler, BroadcastSchedulerFactory, ScoreResponse
+from loopchain.baseservice import BroadcastCommand, BroadcastScheduler, BroadcastSchedulerFactory, \
+    ScoreResponse
```
Comment (metalg0su) [inline loopchain/channel/channel_inner_service.py]: Can you explain why did you break the line?

Comment (windies21) [reply loopchain/channel/channel_inner_service.py]: You got it.

**Comment by windies21** [comment]
test me.

**Comment by windies21** [comment]
please test.

**Pull request merged**